README.md

```diff
@@ -18,90 +16,87 @@ The server will be accesible at `localhost:9000/beeswax/deploy/api/v1`
 A prebuilt docker image is available with:
  
 ```sh
-docker pull jbowen/enrober:v0
+docker pull jbowen/enrober:v0.0.1
 ```
 
 To deploy the server as a docker container on a kubernetes cluster you should use the provided `deploy.yaml` file. Running `kubectl create -f deploy.yaml` will pull the image from dockerhub and deploy it to the default namespace.
 
 The server will be accesible at `<pod-ip>/beeswax/deploy/api/v1`
 
-###Modifying the Server
-
-To build a static binary:
+You may also choose to build the files into your own docker image using the following steps:  
 
 ```sh
+#Build a static binary
 CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' -o enrober .
-```
 
-To build a docker image:
-
-```sh
+#Build a docker image
 docker build -t enrober .
-```
-
-##API Design
 
-The current implementation of the API is simple. It allows for a `GET` request to be made at any of the three levels `namespace`, `application`, `revision` or a `PUT/POST` request to be made at the `revision` level. 
+#Tag the image
+docker tag -f enrober <desired name>
 
-**Inputs:** 
-
-`{namespace}` is a `string`
+#Push to dockerhub
+docker push <desired-name>
+```
 
-`{application}` is a `string`
+The server can now be deployed using:
 
-`{revision}` is a `string`
+```sh
+kubectl run enrober --image=<desired-name>
+```
 
-A `json` body with a corresponding header of `Content-Type: application/json` is required for `PUT/POST` requests. The body has the following contents:
+##API Design
 
-```
-{
-	"PodCount": 			Int,  				//Required 
-	"Image":				string, 			//Optional
-	"ImagePullSecret":		string				//Optional
-	"TrafficHosts": 		[string],			//Required
-	"PublicPaths":  		[string],			//Required 
-	"PathPort": 			Int	,				//Required
-	"EnvVars": { 			string: string,		//Optional
-	    					...,
-				 			...
-	}
-}
-```
+A swagger.yaml file is provided that documents the API per the OpenAPI specification.
 
 ###Usage
 
 > Assuming you are running the server locally
 
-Get all deployments that match a given namespace:
+Create a new namespace group1-env1 with a secret
```

Comment:
It might make sense to explain what this secret is used for.  You could link to: https://github.com/30x/k8s-pods-ingress#security